### ISSUE TITLE
! Restore polska.e-mapa.net

### ISSUE THREAD

**Issue by maro-21** [osmlab/editor-layer-index#1078]
Someone accidentally removed an important imagery for Poland "polska.e-mapa.net Buildings".
I don't know what happened, I can't find it, the file just disappeared. It was this: https://github.com/osmlab/editor-layer-index/commit/f5345a92c7801f1cb4e44346dfeec5d297c5b0d7

Please restore it because this is one of the most used imageries for Poland which we draw buildings from.

**Comment by rbuffat** [comment]
@maro-21 it seems it was removed in https://github.com/osmlab/editor-layer-index/pull/1064 @Asteliks

**Comment by rbuffat** [comment]
This PR was a follow up to https://github.com/osmlab/editor-layer-index/pull/1063 which added https://raw.githubusercontent.com/osmlab/editor-layer-index/gh-pages/sources/europe/pl/Geoportal2Ewidencjabudynk%C3%B3wWMS.geojson

This source is marked as an overlay. 

![poland_buildings](https://user-images.githubusercontent.com/259530/110242253-7bbf1480-7f55-11eb-8a50-70be4c1608d0.png)

**Comment by Asteliks** [comment]
I can confirm everything is as it should be. The layer "polska.e-mapa.net Buildings" was from an old server and should have been an overlay from the beginning. As stated by @rbuffat the same layer from the proper server was introduced in https://github.com/osmlab/editor-layer-index/pull/1063 as [Geoportal 2: Ewidencja budynków WMS](https://raw.githubusercontent.com/osmlab/editor-layer-index/gh-pages/sources/europe/pl/Geoportal2Ewidencjabudynk%C3%B3wWMS.geojson) making the old layer "Geoportal 2: Ewidencja budynków WMS" redundant. Since the old layer became redundant it has been removed in https://github.com/osmlab/editor-layer-index/pull/1064.

TLDR.: everything is fine. @maro-21 didn't see the layer moved to overlays and rushed to report an issue.

Issue can be closed.

**Comment by rbuffat** [comment]
@maro-21 I'm closing this issue as there was no answer for 21 days. Please leave a comment if you feel that the new imagery source is not equivalent and think the old one should be restored.
